strings/memetips.txt

```diff
@@ -12,17 +12,39 @@ Contrary to popular belief, turning off IFF as a Smartgunner does not actually i
 When playing as a Xenomorph, remember to aim for the groin to inflict additional psychological damage to marines.
 Never tell an officer that you're smarter than them - especially if it's true.
 There is no USS Sulaco.
-There is no such thing as an Intel Officer.
+There is no such thing as a Radiotelephone Operator.
 There is no such thing as a techweb.
+There is no such thing as a tip.
+There is no such thing as a Fatty.
+Dropship update is dropping tomorrow.
+Don't forget your ceramic plates.
 Echo Squad does not exist.
+Reqs should always spend all its resources on the Black Market.
 Foxtrot Squad DEFINITELY does not exist.
+Intel Squad is an unicorn.
```

Comment:
```suggestion
Intel Squad is a unicorn.
```

Reply:
Uh wait you're sure? Isn't this correct? It's a vowel

Reply:
What is the difference between "a unicorn" and "an unicorn"? The correct form is “a unicorn.” This is because although “unicorn” begins with a vowel letter, the first sound is a consonant [y], which speakers use “a” before. “An” is only used before words that begin with a vowel sound (“an apple” “an opinion” e.g.).

Reply:
(copypaste from a dictionary site)

Reply:
This is also why you say "an MD" even though M is consonant, in this case it is pronounced as "am-dee"

Reply:
Damn. I never knew. Fair enough